src/app/auth/userlogin/userlogin.component.html

```diff
@@ -1,41 +1,74 @@
-<main id="content" class="usa-section bypass-block-target">
-  <div class="grid-container">
-    <div class="usa-alert usa-alert--error" id="login-error-wrapper" hidden>
-      <div class="usa-alert__body">
-        <div class="usa-alert__text" id="login-error"></div>
-      </div>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="https://designsystem.digital.gov/stylesheets/uswds.min.css">
+</head>
+<body>
+  <div class="button-container">
+    <div class="login-container">
+      <span class="login-text">Sign in with:</span>
+      <a class="login-image" href="javascript:void(0);" onclick="(function() {
```

Comment:
Please move the onclick function into userlogin.component.ts file

Then you can just have:
```
onclick="signIntoLoginGov()"
```
Or whatever function name you come up with